collector/shared/version.go

```diff
@@ -1,42 +0,0 @@
-package collector_shared
```

Comment:
I would like to have a version compare functionality but no nothing "has" to have it now, but are losing a feature we may need later.